dhis-2/dhis-api/src/main/java/org/hisp/dhis/util/SharingUtils.java

```diff
@@ -59,4 +65,14 @@ public static final Sharing generateSharingFromIdentifiableObject( IdentifiableO
         sharing.setDtoUserAccesses( object.getUserAccesses() );
         return sharing;
     }
+
+    public static String withAccess( String jsonb, UnaryOperator<String> accessTransformation )
+        throws JsonProcessingException
+    {
+        ObjectMapper mapper = new ObjectMapper();
```

Comment:
Could this be statically initialized? so we don't need to create a new one each time `withAccess(...)` is used